### PR TITLE
#94 Tickets page implementation

### LINKED ISSUE #94: Implement Tickets Page

### PR DESCRIPTION
**Pull request by Mbabazi12**
### What does this PR do?
This PR is for implementing Tickets page

### Description of Task to be completed
The task was all about designing the tickets page and displaying them if any.

### How This should be tested?
- To test this task after cloning the project on your end, checkout my branch `ft-tickets-page`, and then run the project
- After that go on the sidebar on the tickets page and see if you have any tickets available.
- If you have tickets you can scroll on the pagination and filter to show how many you want || go to page that you want.

### Screenshots

Ticket(Dark)
Ticket(Light)
TicketDetails(Dark)
TicketDetails

### CODE REVIEW

File: package.json
Comment (pacifiquemboni) [inline package.json:1]: @Mbabazi12 may be the issue is comming from these changes made on many dependencies you can check that

File: package.json [hunk 1]
```diff
@@ -33,34 +33,37 @@
...
+    "@expo/vector-icons": "^14.0.2",
     "@react-native-async-storage/async-storage": "1.23.1",
-    "@react-navigation/native": "^6.0.2",
+    "@react-native-picker/picker": "2.9.0",
+    "@react-navigation/native": "^7.0.0",
     "@types/react-native-calendars": "^1.1264.7",
     "apollo-server": "^3.13.0",
     "axios": "^1.7.7",
     "date-fns": "^4.1.0",
-    "expo": "~51.0.37",
```
Comment (niyibi250) [inline package.json:43]: why did you change expo version to 52 , i think we should use single expo version

Comment (Mbabazi12) [reply package.json:43]: 51 is not working i already said that.

Comment (MugemaneBertin2001) [reply package.json:43]: @Mbabazi12  We'll find a way  to make it work on your side without changing mainstream

Comment (Mbabazi12) [reply package.json:43]: @MugemaneBertin2001 once you fix dev i will rebase with it, so for now i had to carry on my task with possible solution.

Comment (MugemaneBertin2001) [reply package.json:43]: It's ok @Mbabazi12 but question is does dev work on your side?
